src/utils.ts

```diff
@@ -28,13 +28,13 @@ export const L1_BRIDGE_ABI = new utils.Interface(require("../abi/IL1Bridge.json"
 export const L2_BRIDGE_ABI = new utils.Interface(require("../abi/IL2Bridge.json"));
 export const NONCE_HOLDER_ABI = new utils.Interface(require("../abi/INonceHolder.json"));
 
-export const ETH_ADDRESS = "0x0000000000000000000000000000000000000000";
+export const LEGACY_ETH_ADDRESS = "0x0000000000000000000000000000000000000000";
 // in the contracts the zero address can not be used, use one instead
 export const ETH_ADDRESS_IN_CONTRACTS = "0x0000000000000000000000000000000000000001";
 export const BOOTLOADER_FORMAL_ADDRESS = "0x0000000000000000000000000000000000008001";
 export const CONTRACT_DEPLOYER_ADDRESS = "0x0000000000000000000000000000000000008006";
 export const L1_MESSENGER_ADDRESS = "0x0000000000000000000000000000000000008008";
-export const L2_ETH_TOKEN_ADDRESS = "0x000000000000000000000000000000000000800a";
```

Comment:
This is also part of public API. In this case I am to mark it as deprecated and use new constant `L2_BASE_TOKEN_ADDRESS` to be more intuitive.